packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -248,11 +258,12 @@ describe('GovernanceWrappedERC20', function () {
   describe('delegate', async () => {
     beforeEach(async function () {
       // approve and deposit for all token holders
-      let promises = defaultBalances.map(balance =>
-        erc20.approve(balance.account, balance.amount)
-      );
-
-      await Promise.all(promises);
+      for (let i = 0; i < defaultBalances.length; i++) {
```

Comment:
see above

Reply:
I know, but Promise.all doesn't work for zksync :)